src/cmake_config.h.in

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file cmake_config.h
+ * @author Radek Krejci <rkrejci@cesnet.cz>
+ * @brief Various variables provided by cmake and compile time options.
+ *
+ * Copyright (c) 2021 CESNET, z.s.p.o.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef LY_CMAKE_CONFIG_H_
+#define LY_CMAKE_CONFIG_H_
+
+/*
+ * Plugins
+ */
+#define LY_PLUGIN_SUFFIX "@CMAKE_SHARED_LIBRARY_SUFFIX@"
+#define LY_PLUGIN_SUFFIX_LEN (sizeof LY_PLUGIN_SUFFIX - 1)
+#define LY_TYPE_PLUGINS_DIR "@PLUGINS_DIR_EXTENSIONS@"
+#define LYEXT_PLUGINS_DIR "@PLUGINS_DIR_TYPES@"
```

Comment:
To make it consistent, why not call it `LY_EXT_PLUGINS_DIR`?

Reply:
All the extension related functions in API are prefixed as `lyext` while the type related have `ly_type` prefix.

Reply:
But yes, here it make sense to unify it as you propose.

Reply:
ok, after looking into the code, I would prefer to keep the current naming which follows the prefix of the extension-related macros and variables.

Reply:
Well, then why is one using the prefix `ly_type` and the other `lyext`? :) I think it is worth it to have it consistent somehow, it decreases the chance of using the wrong name/prefix.

Reply:
let's make such a discussion separated, I'm not going to rename all the extension related API as part of this PR

Reply:
alright